srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/EditBooleanVariableTest.java

```diff
@@ -2,20 +2,50 @@
 
 import static org.hamcrest.Matchers.*;
 import static org.junit.Assert.*;
+import gov.va.med.srcalc.domain.model.*;
 import gov.va.med.srcalc.service.MockModelService;
+import gov.va.med.srcalc.web.view.Views;
 
 import org.junit.Test;
 
 public class EditBooleanVariableTest
 {
+    private final MockModelService fModelService = new MockModelService();
+    
     @Test
     public final void testBasic()
```

Comment:
It might be trivial, but we should test for the getTypeName() in this and upcoming classes.